config.default.yaml

```diff
@@ -25,7 +25,7 @@ enable:
 custom_rules: []  # Default empty [] or link to custom rule file e.g. ["my_folder/my_rules.smk"] that add rules to Snakefile
 
 run:
-  name: "" # use this to keep track of runs with different settings
+  name: "NG_BJ" # use this to keep track of runs with different settings
```

Comment:
Please revert

Reply:
Thanks for the observation. I have reverted this in the recent commits.